### PR TITLE
Add Mailmodo to Segment Doc

### PR DESCRIPTION
**Pull request by sneha-shashidhar**
### Proposed changes

Added the Segment documentation for Mailmodo, post reviewing the Dev Center integration submission. 

### Merge timing

- ASAP once approved

### Related issues (optional)

None

### CODE REVIEW

File: src/connections/destinations/catalog/mailmodo/index.md [hunk 1]
```diff
@@ -1,7 +1,67 @@
...
-title: 'Mailmodo Destination'
-hidden: true
+title: Mailmodo Destination
+rewrite: true
 id: 623a07123d307e60f268a1c2
-published: false
-beta: true
 ---
+
+[Mailmodo](https://www.mailmodo.com/){:target="_blank"} is a powerful email marketing tool focused on bringing the interactive experience of the AMP framework to emails. It allows businesses to create emails with web page-like interactivity right inside the user inbox, thereby increasing engagement and conversions multi-fold.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/mailmodo/index.md:7]:
```suggestion
[Mailmodo](https://www.mailmodo.com/){:target="_blank"} is an email marketing tool that focuses on bringing the interactive experience of the AMP framework to emails. It allows businesses to create emails with web page-like interactivity right inside the user inbox, thereby increasing engagement and conversions multi-fold.
```

File: src/connections/destinations/catalog/mailmodo/index.md [hunk 2]
```diff
@@ -1,7 +1,67 @@
...
+
+
+This destination is maintained by Mailmodo. For any issues with the destination, [contact the Mailmodo Support team](mailto:help@mailmodo.com).
```
Comment (stayseesong) [inline src/connections/destinations/catalog/mailmodo/index.md:10]:
```suggestion
Mailmodo maintains this destination. For any issues with the destination, [contact the Mailmodo Support team](mailto:help@mailmodo.com).
```

File: src/connections/destinations/catalog/mailmodo/index.md [hunk 3]
```diff
@@ -1,7 +1,67 @@
...
+
+
+## Getting Started
+
+{% include content/connection-modes.md %}
+
+1. From the Destinations catalog page in the Segment App, click **Add Destination**.
+2. Search for "Mailmodo" in the Destinations Catalog, and select the "Mailmodo" destination.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/mailmodo/index.md:18]:
```suggestion
2. Search for **Mailmodo** in the Destinations Catalog, and select the **Mailmodo** destination.
```

File: src/connections/destinations/catalog/mailmodo/index.md [hunk 4]
```diff
@@ -1,7 +1,67 @@
...
+3. Choose which Source should send data to the "Mailmodo" destination.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/mailmodo/index.md:19]:
```suggestion
3. Choose which Source should send data to the Mailmodo destination.
```

File: src/connections/destinations/catalog/mailmodo/index.md [hunk 5]
```diff
@@ -1,7 +1,67 @@
...
+4. Go to the [Mailmodo Dashboard](https://manage.mailmodo.com/app/dashboard), navigate to **Settings > API Keys**, then create a new API Key and copy the same.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/mailmodo/index.md:20]:
```suggestion
4. Go to the [Mailmodo Dashboard](https://manage.mailmodo.com/app/dashboard){:target="_blank"} and navigate to **Settings > API Keys**, then create a new API Key and copy it.
```

File: src/connections/destinations/catalog/mailmodo/index.md [hunk 6]
```diff
@@ -1,7 +1,67 @@
...
+5. Enter the "API Key" in the "Mailmodo" destination settings in Segment.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/mailmodo/index.md:21]:
```suggestion
5. Enter the API Key in the Mailmodo destination settings in Segment.
```

File: src/connections/destinations/catalog/mailmodo/index.md [hunk 7]
```diff
@@ -1,7 +1,67 @@
...
+
+
+## Supported methods
+
+Mailmodo supports the following methods, as specified in the [Segment Spec](/docs/connections/spec).
+
+### Identify
+
+If you aren’t familiar with the Segment Spec, take a look at the [Identify method documentation](/docs/connections/spec/identify) to learn about what it does.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/mailmodo/index.md:30]:
```suggestion
Take a look at the [Identify method documentation](/docs/connections/spec/identify) to learn about what it does.
```

File: src/connections/destinations/catalog/mailmodo/index.md [hunk 8]
```diff
@@ -1,7 +1,67 @@
...
+analytics.identify('userId12345', {
+  firstName: 'Bob',
+  lastName: 'Dole',
+  email: 'bob.dole@example.com',
+  company: 'Initech',
+  employees: 234
+});
+```
+
+Every time you call identify with an email address included, Mailmodo:
```
Comment (stayseesong) [inline src/connections/destinations/catalog/mailmodo/index.md:42]:
```suggestion
Every time you make an identify call with an email address included:
```

File: src/connections/destinations/catalog/mailmodo/index.md [hunk 9]
```diff
@@ -1,7 +1,67 @@
...
+1.	Verifies that the email exists.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/mailmodo/index.md:43]:
```suggestion
1.	Mailmodo verifies that the email exists.
```

File: src/connections/destinations/catalog/mailmodo/index.md [hunk 10]
```diff
@@ -1,7 +1,67 @@
...
+2.	If the email doesn’t exist, Mailmodo adds the user as a Contact to the Mailmodo database and matches user properties with the Segment `traits` sent in identify call payload.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/mailmodo/index.md:44]:
```suggestion
2.	If the email doesn’t exist, Mailmodo adds the user as a contact to the Mailmodo database and matches user properties with the Segment `traits` sent in the identify call payload.
```

File: src/connections/destinations/catalog/mailmodo/index.md [hunk 11]
```diff
@@ -1,7 +1,67 @@
...
+3.	If the email exists, Mailmodo updates the user properties for the Contact against the Segment `traits` sent in identify call payload.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/mailmodo/index.md:45]:
```suggestion
3.	If the email exists, Mailmodo updates the user properties for the Contact against the Segment `traits` sent in the identify call payload.
```

File: src/connections/destinations/catalog/mailmodo/index.md [hunk 12]
```diff
@@ -1,7 +1,67 @@
...
+
+All the [special traits](/docs/connections/spec/identify#traits) recognized by Segment will be translated and matched with the Mailmodo user properties for a Contact. These fields will be automatically created or mapped for a Contact in Mailmodo and will be available for personalization and advance segmentation.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/mailmodo/index.md:47]:
```suggestion
All the [special traits](/docs/connections/spec/identify#traits) recognized by Segment are translated and matched with the Mailmodo user properties for a Contact. These fields are automatically created or mapped for a Contact in Mailmodo and are available for personalization and advance segmentation.
```

File: src/connections/destinations/catalog/mailmodo/index.md [hunk 13]
```diff
@@ -1,7 +1,67 @@
...
+
+Every time you make an identify call with an email address included:
+1.	Verifies that the email exists.
+2.	If the email doesn’t exist, Mailmodo adds the user as a Contact to the Mailmodo database and matches user properties with the Segment `traits` sent in identify call payload.
+3.	If the email exists, Mailmodo updates the user properties for the Contact against the Segment `traits` sent in identify call payload.
+
+All the [special traits](/docs/connections/spec/identify#traits) recognized by Segment will be translated and matched with the Mailmodo user properties for a Contact. These fields will be automatically created or mapped for a Contact in Mailmodo and will be available for personalization and advance segmentation.
+
+> info "How Mailmodo handles incoming email addresses"
+> 1. The email field is required. Identify calls without an email are dropped.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/mailmodo/index.md:50]:
```suggestion
> * The email field is required. Mailmodo drops identify calls without an email.
```

File: src/connections/destinations/catalog/mailmodo/index.md [hunk 14]
```diff
@@ -1,7 +1,67 @@
...
+> 2. If different email addresses are sent against same user id in identify call, Mailmodo treats them as two different contacts.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/mailmodo/index.md:51]:
```suggestion
> * If different email addresses are sent against the same user ID in identify call, Mailmodo treats them as two different contacts.
```

File: src/connections/destinations/catalog/mailmodo/index.md [hunk 15]
```diff
@@ -1,7 +1,67 @@
...
+
+### Track
+If you aren’t familiar with the Segment Spec, take a look at the [Track method documentation](/docs/connections/spec/track) to learn about what it does. An example call would look like:
```
Comment (stayseesong) [inline src/connections/destinations/catalog/mailmodo/index.md:54]:
```suggestion
Take a look at the [Track method documentation](/docs/connections/spec/track) to learn about what it does. An example call looks like:
```

File: src/connections/destinations/catalog/mailmodo/index.md [hunk 16]
```diff
@@ -1,7 +1,67 @@
...
+```js
+analytics.track('Product Viewed', {
+  product_id: '507f1f77bcf86cd799439011',
+  name: 'Monopoly: 3rd Edition',
+  price: 18.99,
+  url: 'https://www.example.com/product/path',
+  image_url: 'https://www.example.com/product/path.jpg'
+});
+```
+Segment sends `Track` calls to Mailmodo as a Custom Event. When you call  track, Segment sends the event to Mailmodo with the event name and all properties that you specified.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/mailmodo/index.md:65]:
```suggestion
Segment sends `Track` calls to Mailmodo as a Custom Event. When you call track, Segment sends the event to Mailmodo with the event name and all properties that you specified.
```

File: src/connections/destinations/catalog/mailmodo/index.md [hunk 17]
```diff
@@ -1,7 +1,67 @@
...
+
+Be sure you send an Identify call for any user who will trigger Track calls. If Mailmodo receives a Track call for an unknown userId, the call is dropped.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/mailmodo/index.md:67]:
```suggestion
Be sure you send an Identify call for any user who triggers Track calls. If Mailmodo receives a Track call for an unknown userId, the call drops.
```

**Review by stayseesong — CHANGES_REQUESTED** [review]
just a couple of suggested edits

**Review by markzegarelli — APPROVED** [review]

**Review by stayseesong — APPROVED** [review]

**Pull request merged**